jest.config.json

```diff
@@ -0,0 +1,14 @@
+{
+	"clearMocks": true,
+	"collectCoverage": true,
+	"collectCoverageFrom": [
+		"**/*.js"
```

Comment:
suggestion: Maybe we should leave test files out the coverage, adding `"!test/**/*.js"`? Or in the coveragePathIgnorePatterns...

Reply:
I can't find the documentation for it at all, but Jest/Istanbul already seems to ignore any files that it uses as part of the tests? I might add this for safety anyway, but it seems fine locally:

<img width="583" alt="Screenshot 2022-05-03 at 11 38 28" src="https://user-images.githubusercontent.com/138944/166439642-2e093b05-6bc4-4d11-961e-77fc9067ee12.png">

Reply:
OK I did this because this won't account for things like mocks and fixture files 👍